invalid memory address or nil pointer dereference

I'm unable to connect with SSL enabled gRPC server using evans

https://github.com/simplesteph/grpc-go-course/blob/master/ssl/instructions.sh
I followed the steps above to generate certificates. The server started successfully after loading the certificates generated.

Below is my server code

`
import io.grpc.ServerBuilder;
import user.UserService;

import java.io.File;
import java.io.IOException;

import io.grpc.Server;
import io.grpc.protobuf.services.ProtoReflectionService;

public class gRPCService {
	public static void main (String [] args) throws IOException, InterruptedException {
		
		try {
			File cert = new File("ssl/server.crt");
			File key = new File("ssl/server.pem");
			
			Server server = ServerBuilder.forPort(9091)
					.useTransportSecurity(cert, key)
					.addService(ProtoReflectionService.newInstance())
					.addService(new UserService())
					.build()
					.start();
			
			System.out.println("Server started at " + server.getPort());
			
			server.awaitTermination();
		} catch (Exception e) {
			System.out.println("Server started at " + e);
			e.printStackTrace();
		}
	}
}
`

C:\gRPC\ssl>evans -p 9091 -r -t --cacert ca.crt --certkey server.key --cert server.csr --servername localhost
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x10 pc=0x8b8c3d]

goroutine 1 [running]:
github.com/ktr0731/evans/idl/proto.LoadByReflection(0x0, 0x0, 0x0, 0x0, 0x0, 0x1000000)
        /go/src/github.com/ktr0731/evans/idl/proto/proto.go:155 +0x1d
github.com/ktr0731/evans/mode.newSpec(0x12190900, 0x0, 0x0, 0x1, 0x1, 0x12208980, 0x12185890)
        /go/src/github.com/ktr0731/evans/mode/common.go:16 +0x5e
github.com/ktr0731/evans/mode.RunAsREPLMode(0x12190900, 0xbebd00, 0x121cb588, 0x121857d0, 0x0, 0x0)
        /go/src/github.com/ktr0731/evans/mode/repl.go:28 +0xe9
github.com/ktr0731/evans/app.(*App).run(0x12004ff0, 0x12062008, 0xc, 0xf, 0x0, 0x0)
        /go/src/github.com/ktr0731/evans/app/app.go:123 +0x357
github.com/ktr0731/evans/app.(*App).Run(0x12004ff0, 0x12062008, 0xc, 0xf, 0x12004fe0)
        /go/src/github.com/ktr0731/evans/app/app.go:40 +0x3d
main.main()
        /go/src/github.com/ktr0731/evans/main.go:11 +0xac

--- a/config/testdata/fixtures/create_a_default_global_config_if_both_of_global_and_local_config_are_not_found.golden.toml
+++ b/config/testdata/fixtures/create_a_default_global_config_if_both_of_global_and_local_config_are_not_found.golden.toml
@@ -10,7 +10,7 @@
 
 [meta]
   autoupdate = false
-  configversion = "0.8.0"
+  configversion = "0.8.1"
   updatelevel = "patch"
 
 [repl]

--- a/meta/meta.go
+++ b/meta/meta.go
@@ -5,5 +5,5 @@ import version "github.com/hashicorp/go-version"
 const AppName = "evans"
 
 var (
-	Version = version.Must(version.NewSemver("0.8.0"))
+	Version = version.Must(version.NewSemver("0.8.1"))
 )

--- a/mode/repl.go
+++ b/mode/repl.go
@@ -12,22 +12,19 @@ import (
 	"github.com/ktr0731/evans/prompt"
 	"github.com/ktr0731/evans/repl"
 	"github.com/ktr0731/evans/usecase"
-	"github.com/ktr0731/go-multierror"
 	"github.com/pkg/errors"
 )
 
 func RunAsREPLMode(cfg *config.Config, ui cui.UI, cache *cache.Cache) error {
-	var result error
 	gRPCClient, err := newGRPCClient(cfg)
 	if err != nil {
-		result = multierror.Append(result, err)
-	} else {
-		defer gRPCClient.Close(context.Background())
+		return errors.Wrap(err, "failed to instantiate a new gRPC client")
 	}
+	defer gRPCClient.Close(context.Background())
 
 	spec, err := newSpec(cfg, gRPCClient)
 	if err != nil {
-		result = multierror.Append(result, err)
+		return errors.Wrap(err, "failed to instantiate a new spec")
 	}
 
 	usecase.Inject(
@@ -36,10 +33,6 @@ func RunAsREPLMode(cfg *config.Config, ui cui.UI, cache *cache.Cache) error {
 		gRPCClient,
 		json.NewPresenter(),
 	)
-
-	if result != nil {
-		return result
-	}
 
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()

